google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/Spanner.java

```diff
@@ -26,14 +26,45 @@
  */
 public interface Spanner extends Service<SpannerOptions> {
   /** Returns a {@code DatabaseAdminClient} to do admin operations on Cloud Spanner databases. */
+  /*
+   * <!--SNIPPET get_dbadmin_client-->
+   * <pre>{@code
+   * SpannerOptions options = SpannerOptions.newBuilder().build();
+   * Spanner spanner = options.getService();
+   * DatabaseAdminClient dbAdminClient = spanner.getDatabaseAdminClient();
+   * }</pre>
+   * <!--SNIPPET get_dbadmin_client-->
+   */
   DatabaseAdminClient getDatabaseAdminClient();
 
   /** Returns an {@code InstanceAdminClient} to do admin operations on Cloud Spanner instances. */
+  /*
+   * <!--SNIPPET get_instance_admin_client-->
+   * <pre>{@code
+   * SpannerOptions options = SpannerOptions.newBuilder().build();
+   * Spanner spanner = options.getService();
+   * InstanceAdminClient instanceAdminClient = spanner.getInstanceAdminClient();
+   * }</pre>
+   * <!--SNIPPET get_instance_admin_client-->
+   */
   InstanceAdminClient getInstanceAdminClient();
 
   /**
    * Returns a {@code DatabaseClient} for the given database. It uses a pool of sessions to talk to
    * the database.
+   *
+   * <!--SNIPPET get_db_client-->
+   * <pre>{@code
+   * SpannerOptions options = SpannerOptions.newBuilder().build();
+   * Spanner spanner = options.getService();
+   * final String project = "test-project";
+   * final String instance = "test-instance";
+   * final String database = "example-db";
+   * DatabaseId db =
+   *     DatabaseId.of("span-cloud-testing", "nsujir-ins", "example-db");
```

Comment:
use generic examples..

Reply:
Will fix. Note that this change is merged so all this will have to be a separate change.